### PR TITLE
Md 16 respond to background

### LINKED ISSUE #25: Make content script respond to camera messages
The messages the content script needs to respond to are:
* `StartCapture`
* `EndCapture`
* `GetFrame`

### LINKED ISSUE #26: Make content script respond to screen modification messages
The content script needs to respond to the following messages:
* `ShowCalibrateScreen`
* `HideCalibrateScreen`
* `Blur`
* `Unblur`

### PR DESCRIPTION
**Pull request by ThatOneGoat**
Closes #25
Closes #26

### CODE REVIEW

File: js/content.js [hunk 1]
```diff
@@ -1,58 +1,117 @@
...
+});
+
+function setupVid() {
+    // setting up video element
+    video.autoplay = true;
+    //video.style.display = "none";
+    video.style.width = "450px";
+    video.style.height = "450px";
+    video.style.borderRadius = "350px";
+    video.style.zIndex = "21434536743436566";
```
Comment (mmattikalli) [inline js/content.js]: This index needs to be 1000000001 exactly to align with everything else

File: js/content.js [hunk 2]
```diff
@@ -61,17 +120,14 @@ function addBlur() {
     document.body.appendChild(newWebsiteDiv);
     newWebsiteDiv.innerHTML = websiteBody;
 
-    // sets document zindex lower than the divContainer     
+    // sets document zindex lower than the divContainer
     document.body.style.zIndex = "10000";
 
-    // creates video element 
-    let video = document.createElement("video");
```
Comment (mmattikalli) [inline js/content.js]: where is video created

Comment (Kaboomboom3) [reply js/content.js]: Video is created on line 4

File: js/popup.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ document.addEventListener("DOMContentLoaded", () => {
     faceCheckbox.addEventListener('click', () => {
         browser.runtime.sendMessage({ type: 'IsLockEnabled' }, enabled => {
             if (enabled) {
+                console.log("ooooooooooo");
```
Comment (mmattikalli) [inline js/popup.js]: Unnecessary (?)

**Pull request merged**